### DISCUSSION TITLE
Downloading the secutiryonion .iso my antivirus detects a malware

### DISCUSSION THREAD

**Discussion by dcantos1** [Security-Onion-Solutions/securityonion#6051] [General] [answered]
Hello everyone, I am downloading the .ISO to carry out a laboratory in my environment but my antivirus detects the .ISO with highly dangerous malware and I am worried that it contains some malware camouflaged in the solution. attached image

![malware](https://user-images.githubusercontent.com/80493416/139446470-d002598d-0672-4d9d-9484-4954e2f6bff7.jpg)

![malware2](https://user-images.githubusercontent.com/80493416/139446677-c1cba8b8-0403-4e2b-9d30-f80cb094bc11.png)

**Comment by Acewiza** [comment]
Don't recall exactly when it went up, but that .iso's been around awhile now.  False positive seems like a good bet on the trojan alert.

**Comment by jertel** [comment, marked as answer]
See #4532, #4306, and #4235.
